lib/linguist/heuristics.yml

```diff
@@ -113,6 +113,12 @@ disambiguations:
     #  : dependency
     # path/file.ext1 : some/path/../file.ext2
     pattern: '([\/\\].*:\s+.*\s\\$|: \\$|^[ %]:|^[\w\s\/\\.]+\w+\.\w+\s*:\s+[\w\s\/\\.]+\w+\.\w+)'
+- extensions: ['.dsp']
+  rules:
+  - language: Microsoft Developer Studio Project
+    pattern: '# Microsoft Developer Studio Generated Build File'
```

Comment:
Tabs are awesome and everybody should use them, but I'm afraid they're invalid in YAML; these lines need to use spaces for indentation (much as I hate to say it…)

Reply:
Hi @Alhadis,
I do not understand the remark; I have checked the file, and it does not contain any tab character. I don't use them usually and I try my best to adapt, and in this case I cannot see where exactly is the problem here.

Reply:
Crap, I forgot to disable a browser extension. Nevermind. 😂